hugegraph-core/src/main/java/org/apache/hugegraph/util/StringEncoding.java

```diff
@@ -14,19 +14,22 @@
 
 package org.apache.hugegraph.util;
 
-import java.io.UnsupportedEncodingException;
+import static java.nio.charset.StandardCharsets.UTF_8;
```

Comment:
don't import static as possible,  import StandardCharsets instead, and can define a static final UTF8 = StandardCharsets.UTF_8